File: samples_generated_code/sample_3/phi4/baseline_code_py/code_row_109.py
import random

def getMonthlySales():
    return random.randint(0, 100000)

# Initialize quarterly sales accumulators
quarter1_sales = 0
quarter2_sales = 0
quarter3_sales = 0
quarter4_sales = 0

# Iterate over each month and accumulate sales per quarter
for month in range(1, 13):
    monthly_sales = getMonthlySales()
    
    if month <= 3:
        quarter1_sales += monthly_sales
    elif month <= 6:
        quarter2_sales += monthly_sales
    elif month <= 9:
        quarter3_sales += monthly_sales
    else:
        quarter4_sales += monthly_sales

# Print the results for each quarter
print(f"Quarter 1 Sales: {quarter1_sales}")
print(f"Quarter 2 Sales: {quarter2_sales}")
print(f"Quarter 3 Sales: {quarter3_sales}")
print(f"Quarter 4 Sales: {quarter4_sales}")

File: samples_generated_code/sample_3/phi4/baseline_code_java/code_row_109.java
import java.util.Random;

public class MonthlySales {

    public static void main(String[] args) {
        int q1 = 0, q2 = 0, q3 = 0, q4 = 0;
        
        for (int month = 1; month <= 12; month++) {
            int sales = getMonthlySales();
            
            switch (month) {
                case 1:
                case 2:
                case 3:
                    q1 += sales;
                    break;
                case 4:
                case 5:
                case 6:
                    q2 += sales;
                    break;
                case 7:
                case 8:
                case 9:
                    q3 += sales;
                    break;
                case 10:
                case 11:
                case 12:
                    q4 += sales;
                    break;
            }
        }
        
        System.out.println("Quarter 1 Sales: " + q1);
        System.out.println("Quarter 2 Sales: " + q2);
        System.out.println("Quarter 3 Sales: " + q3);
        System.out.println("Quarter 4 Sales: " + q4);
    }

    public static int getMonthlySales() {
        Random random = new Random();
        return random.nextInt(100001); // Generates a number between 0 and 100000
    }
}